lib/PuppeteerSharp/Page.cs

```diff
@@ -169,6 +170,10 @@ private Page(Session client, Target target, FrameTree frameTree, bool ignoreHTTP
                 {
                     cookie.Url = Url;
                 }
+                if (cookie.Url == "about:blank")
+                {
+                    throw new PuppeteerException($"Blank page can not have cookie \"{cookie.Name}\"");
+                }
```

Comment:
According to [Puppeteer's test](https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/test/test.js#L3253). We should let Chromium fail and throw an exception from the connection https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/lib/Connection.js#L98